### ISSUE TITLE
sphereface2 loss with AAM pretrained

### ISSUE THREAD

**Issue by EmreOzkose** [wenet-e2e/wespeaker#183]
Hi,

Dou you think that it will be a good option to finetune pretrained Resnet models (with AAM loss) with sphereface2 ? Do you have any experience like that?

**Comment by JiJiJiang** [comment]
@Hunterhuan
